### PR TITLE
Building with Responsive Design - Milestones 2

### PR DESCRIPTION
**Pull request by Maha-Magdy**
work that has done in this milestone
- adding more three sections Latest news, Categories, Latest deals
- working on responsive of the new three sections
- update read me file with upon the new partner

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2,455 +2,1416 @@
...
+    </div>
+    <ul class="latest-news-container mt-20 mb-50">
+      <li class="latest-new d-flex mb-10">
+        <a href="#" class="decoration-none">
+          <div class="latest-news-img latest-news-img1 mr-10"></div>
+        </a>
+        <div class="latest-news-text">
+          <a href="#" class="latest-news-title decoration-none d-block fw-bold">
+            Want to drive a hydrogen EV in the US? Move to California
+          </a>
```
Comment (wathigo) [inline index.html]: The article's title should be in (hx) tag and a link (a) tag.

File: index.html [hunk 2]
```diff
@@ -2,455 +2,1416 @@
...
+        <li class="latest-new d-flex mb-10">
+          <a href="#" class="decoration-none">
+            <div class="latest-deals-img latest-deals-img1 mr-10 less"></div>
+          </a>
+          <div class="story-dealDiscount">
+            <div>97% off</div>
+        </div>
+          <div class="latest-news-text mt-10">
+            <a href="#" class="latest-deals-title decoration-none d-block fw-bold">
+              The Ultimate 2021 White Hat Hacker Certification Bundle
```
Comment (wathigo) [inline index.html:1229]: The title of the section has a header tag (`hx`) and a link (`a` tag)

File: index.html [hunk 3]
```diff
@@ -2,455 +2,1416 @@
...
+      <div class="general-title">
+        <h2 class="mt-50 latest-news-main-title">
+          <a class="decoration-none" href="#">
+            Latest deals
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16">
+              <path fill="none" stroke="#F42" stroke-width="3" stroke-miterlimit="10" d="M4.75 1.5l6.5 6.5-6.5 6.5" />
+            </svg>
+          </a>
+        </h2>
+      </div>
```
Comment (wathigo) [inline index.html]: The link (`a` tag) to "See all deals" is in the same row that the title but aligned to the right.
Let's include the link on the right. 
Screenshot from 2021-02-25 14-34-42

**Review by wathigo — CHANGES_REQUESTED** [review]

**Review by wathigo — CHANGES_REQUESTED** [review]
## CHANGES REQUIRED :o: 
Great job on this milestone.
Work done so far demonstrates mastery in the following concepts:

* Use of semantic HTML 5 elements.
* Use of media queries and mobile-first approach to make a page responsive.
* Use of flex/grid CSS properties.

The files are well organized and a mobile-first approach has been implemented.
Let's fix a few issues before we move on. See the comments :point_up: : 

**After making the changes, submit another request for review.**

**Happy Coding :computer: 
Simon Wathigo**

File: index.html (hunk 3 above)
Comment (Maha-Magdy) [reply index.html]: Okay, thanks for your review. we'll work on them.

**Review by wathigo — APPROVED** [review]
## APPROVED :heavy_check_mark: 
Great job working on the suggested changes.


**You may move on**

**Happy Learning :technologist: 
Simon Wathigo**

**Pull request merged**